src/index.html

```diff
@@ -5,13 +5,94 @@
     <title>Webpack Starter Kit</title>
   </head>
   <body>
-    <article class="diplay-box">
-      <form>
-        <input id="username">
-        <input id="password">
-        <button type="button" onclick="logIn()">click me</button>
+
+    <header>
+    <h1>Travel Tracker</h1>
+    <p class="amount-spent">Amount spent on travel this year: $1000</p>
+    </header>
+    <article class="display-box">
+      <form class="entry-form">
+        <section class="login-area">
+          <label for="username">Username:</label>
+          <input id="username">
+          <label for="password">Password:</label>
+          <input id="password" type="password">
+        </section>
+        <button class="login-button" type="button">Log In</button>
       </form>
     </article>
+    <article class="trip-input-box hidden">
+      <form class="entry-form">
+        <section class="login-area">
+          <label for="date">Date:</label>
+          <input id="date">
+          <label for="duration">Duration:</label>
+          <input id="duration">
+          <label for="num-travelers">Number of Travelers:</label>
+          <input id="num-travelers">
+          <label for="destination">Destination:</label>
+          <!-- <input id="destination"> -->
+          <select name="cars" id="cars">
+            <option value="lima">Lima, Peru</option>
+            <option value="stockholm">Stockholm, Swedwen</option>
+            <option value="sydney">Sydney, Austrailia</option>
+            <option value="cartagena">Cartagena, Columbia</option>
+            <option value="madrid">Madrid, Spain</option>
+            <option value="jakarta">Jakarta, Indonesia</option>
+            <option value="paris">Paris, France</option>
+            <option value="tokyo">Tokyo, Japan</option>
+            <option value="amsterdam">Amsterdam, Netherlands</option>
+            <option value="toronto">Toronto, Japan</option>
+
+            <option value="mikonos">Mikonos, Greece</option>
+            <option value="wellington">Wellington, New Zealand</option>
+            <option value="petersburg">St. Petersburg, Russia</option>
+            <option value="marrakesh">Marrakesh, Morocco</option>
+            <option value="manila">Manila, Philippines</option>
+            <option value="bangkok">Bangkok, Thailand</option>
+            <option value="jaipur">Jaipur, India</option>
+            <option value="cape">Cape Town, South Africa</option>
+            <option value="quito">Quito, Ecuador</option>
+            <option value="miami">Miami, Florida</option>
```

Comment:
Interesting to see that the HTML for the destinations has been built out by hand. There is a mis-spelling here, such as line 45. Would definitely double check and ensure that everything is correctly entered. Can also consider an alternative option of injecting the select through JS based on the destinations API data.